inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp

```diff
@@ -810,7 +732,7 @@ void MKLDNNNormalizeNode::initSupportedPrimitiveDescriptors() {
             inputPrecision = outputPrecision = Precision::BF16;
     }
 
-    auto isOneOf = [&](InferenceEngine::Precision precision, std::vector<InferenceEngine::Precision> precisions) {
+    auto isOneOf = [&](Precision precision, std::vector<Precision> precisions) {
```

Comment:
could you please reuse `one_of` routine from general_utils.h?

Reply:
reused